docs/conf.py

```diff
@@ -55,9 +55,9 @@
 # the built documents.
 #
 # The short X.Y version.
-version = '0.20.1'
+version = '0.21.0-SNAPSHOT'
 # The full version, including alpha/beta/rc tags.
-release = '0.20.1'
+release = '0.21.0-SNAPSHOT'
```

Comment:
I'm unsure if this is automatically updated if a release is triggered ❓

Reply:
I think it is once the release wf is triggered